third_party/txt/src/txt/font_collection.cc

```diff
@@ -150,11 +150,13 @@ FontCollection::GetMinikinFontCollectionForFamilies(
   }
   // Search for default font family if no user font families were found.
   if (minikin_families.empty()) {
-    const auto default_font_family = GetDefaultFontFamily();
-    std::shared_ptr<minikin::FontFamily> minikin_family =
-        FindFontFamilyInManagers(default_font_family);
-    if (minikin_family != nullptr) {
-      minikin_families.push_back(minikin_family);
+    const auto default_font_families = GetDefaultFontFamilies();
+    for (auto family : default_font_families) {
+      std::shared_ptr<minikin::FontFamily> minikin_family =
+          FindFontFamilyInManagers(family);
+      if (minikin_family != nullptr) {
+        minikin_families.push_back(minikin_family);
+      }
```

Comment:
Exit the loop here.  It should be sufficient to provide the first available default font instead of merging all available default fonts.

Reply:
Oh, good point! Done.